scss/components/counter.scss

```diff
@@ -16,6 +16,7 @@ $block: #{$fd-namespace}-counter;
   --fd-counter-color: var(--fd-color-text-5);
   --fd-counter-background-color: var(--fd-color-status-4);
 
+  @include fd-reset();
```

Comment:
The line above `$fd-counter-font-size: 0.785714285714286em !default;` should be changed to use `px` otherwise you can get different size text between self-contained and "all" styels.  There will be a larger discussion/story about the use of `px` vs `rem` or `em` in the future, but right now, the self-contained text is larger than the "all" styles.

Reply:
I think the counter font-size was initial in `em` as dependant to the parent element. Let's just keep it as it was and then we can tackle it later on